i18n/ru.json

```diff
@@ -155,5 +158,12 @@
     "Enter password":
         "Введите пароль",
     "Uploading paste… Please wait.":
-        "Отправка записи… Пожалуйста подождите."
+        "Отправка записи... Пожалуйста подождите.",
+    "Loading…": "Загрузка...",
+    "Decrypting paste…": "Расшифровка записи...",
+    "Preparing new paste…": "Подготовка новой записи...",
+    "In case this message never disappears please have a look at <a href=\"https://github.com/PrivateBin/PrivateBin/wiki/FAQ#why-does-not-the-loading-message-go-away\">this FAQ for information to troubleshoot</a>.":
+        "Если данное сообщение не исчезает длительное время, посмотрите <a href=\"https://github.com/PrivateBin/PrivateBin/wiki/FAQ#why-does-not-the-loading-message-go-away\">этот FAQ с информацией о возможном решении проблемы</a>.",
```

Comment:
Maybe add the equivalent of "(in English)" after the link to make users aware that the FAQ is written in English.